tools/templates/DOCKING_SOFTWARE_NOTES.md

```diff
@@ -0,0 +1,159 @@
+
+### Running with ADFR: 
+For running ADFR, a trg receptor file (generated during pre-processing) needs to be provided by the user. The config.txt file
+can contain the line: 
+    receptor=test.tmp
+Note: ADFR needs to be installed such that it can run by using the word 'adfr' in shell (from any location).
+
+
+### Running with PLANTS:
+A PLANTS executable (with the exact name PLANTS) must be downloaded from http://www.tcd.uni-konstanz.de/research/plants.php, and 
+placed within the directory /tools/bin (where other executables are located).
+A config.txt file is required for running PLANTS. An example of the config file format is: 
+    protein_file receptor.mol2
```

Comment:
For code blocks in markdown where you want the formatting to stay the same, you should use

\`\`\`
code
\`\`\`

Reply:
Good point! Will update